README.md

```diff
@@ -1,160 +1,132 @@
-McGuire, and many others who were involved in one way or another and
-whose efforts are appreciated.
+### Manually Loading Extension
+
+#### Firefox
+
+1. Open Firefox and navigate to [about:debugging#/runtime/this-firefox/](about:debugging#/runtime/this-firefox/)
+1. Click on 'Load Temporary Add-on' button.
+1. Select `manifest.json` from the `dist` folder.
+1. Check extension logo appears in the top-right corner of the browser.
+
+#### Chrome
+
+1. Open Chrome and navigate to [chrome://extensions/](chrome://extensions/)
+1. Turn on the 'Developer mode' on the top-right corner.
+1. Click on 'Load unpacked' button.
+1. Select the `dist` folder.
+1. Check extension logo appears in the top-right corner of the browser.
+1. If you cannot see the extension logo, it's likely not pinned to the toolbar.
+
+#### Edge
+
+-   Open Edge and navigate to [edge://extensions/](edge://extensions/)
+-   Turn on the 'Developer mode' on the left bar.
+-   Click on 'Load unpacked' button in the main panel.
+-   Select the `dist` folder.
+-   The extension will appear listed in the main panel.
+-   To see the extension in the bar, click in the puzzle icon and enable it, so it gets pinned to the toolbar.
+---
+
+### Highlights
+
+**2018** -- The Privacy Pass protocol is based on a _Verifiable, Oblivious Pseudorandom Function_ (VOPRF) first established by [Jarecki et al. 2014](https://eprint.iacr.org/2014/650.pdf). The details of the protocol were published at [PoPETS 2018](https://doi.org/10.1515/popets-2018-0026) paper authored by Alex Davidson, Ian Goldberg, Nick Sullivan, George Tankersley, and Filippo Valsorda.
+
+**2019** -- The captcha provider [hCaptcha](https://www.hcaptcha.com/privacy-pass) announced support for Privacy Pass.
+
+**2020** -- The CFRG (part of IRTF/IETF) started a [working group](https://datatracker.ietf.org/wg/privacypass/about/) seeking for the standardization of the Privacy Pass protocol.
+
+**2021** -- In this [blog post](https://blog.cloudflare.com/privacy-pass-v3), we announced the [v3](https://github.com/privacypass/challenge-bypass-extension/tree/v3.0.0) version of this extension, which makes the code base more resilient, extensible, and maintainable.
+
+**2022** -- The Privacy Pass protocol can also use RSA blind signatures.
+
+
+#### Acknowledgements
+
+The Privacy Pass team would like to thank Eric Tsai for creating the logo and extension design, Dan Boneh for helping us develop key parts of the protocol, as well as Peter Wu and Blake Loring for their helpful code reviews. We would also like to acknowledge Sharon Goldberg, Christopher Wood, Peter Eckersley, Brian Warner, Zaki Manian, Tony Arcieri, Prateek Mittal, Zhuotao Liu, Isis Lovecruft, Henry de Valence, Mike Perry, Trevor Perrin, Zi Lin, Justin Paine, Marek Majkowski, Eoin Brady, Aaran McGuire, Suphanat Chunhapanya, Armando Faz Hernández, and many others who were involved in one way or another and whose efforts are appreciated.
```

Comment:
The section mentioning the original team has been removed. It should be added back.

```
The creation of the Privacy Pass protocol was a joint effort by the team made up of George Tankersley, Ian Goldberg, Nick Sullivan, Filippo Valsorda, and Alex Davidson.
```